traits/trait_types.py

```diff
@@ -1012,12 +1011,11 @@ class Constant(TraitType):
 
     Traits of this type are very space efficient (and fast) because
     *value* is not stored in each instance using the trait, but only in
-    the trait object itself. The *value* cannot be a list or dictionary,
-    because those types have mutable values.
+    the trait object itself.
 
     Parameters
     ----------
-    value : any type other than list or dict
+    value : Any
```

Comment:
Let's spell this "any" rather than "Any" (as a rule, we don't use Trait types in numpydoc type annotations; we could consider using `typing` types, but I think that's going to get messy and confusing within Traits).